src/playitem.js

```diff
@@ -1,16 +1,26 @@
 var setupAutoadvance = require('./autoadvance.js');
 
+var Cue = window.VTTCue || window.TextTrackCue;
+
 var clearTracks = function(player) {
-  var remoteTT = player.remoteTextTracks();
-  var i = (remoteTT && remoteTT.length) || 0;
+  var
+    remoteTT = player.remoteTextTracks(),
+    ttl = player.textTracks(),
+    i = (remoteTT && remoteTT.length) || 0,
+    j = (ttl && ttl.length) || 0;
 
   while (i--) {
     player.removeRemoteTextTrack(remoteTT[i]);
   }
+
+
+  while (j--) {
+    ttl.removeTrack_(ttl[j]);
```

Comment:
This is not safe and we shouldn't do it. At best, we should set the `mode` of these to `disabled`.